src/main/java/org/jitsi/jicofo/bridge/Bridge.java

```diff
@@ -343,9 +363,35 @@ public String getRegion()
     public String toString()
     {
         return String.format(
-                "Bridge[jid=%s, relayId=%s, region=%s]",
+                "Bridge[jid=%s, relayId=%s, region=%s, stress=%.2f]",
                      jid.toString(),
                      getRelayId(),
-                     getRegion());
+                     getRegion(),
+                     getStress());
+    }
+
+    /**
+     * Returns the "stress" of the bridge. The stress is computed based on the
+     * total bitrate reported by the bridge and the video stream estimation
+     * since the last update from the bridge.
+     *
+     * @return the sum of the last total reported bitrate (in kbps) and an
+     * estimation of the bitrate of the streams that we estimate that the bridge
+     * hasn't reported to Jicofo yet. The estimation is the product of the
+     * number of unreported streams and a constant C (which we set to 1000 kbps).
+     */
+    public double getStress()
+    {
+        return Math.min(MAX_TOTAL_BITRATE_BPS,
```

Comment:
I'm confused by the scale between these two things we're taking the minimum of.  Isn't the goal of this to return a number between 0 and 1?  But the first arg is  a maximum total bitrate in bps?

Reply:
@bbaldino I'm just making sure the number doesn't become greater than 1 (in case our bitrate estimation is greater than MAX_TOTAL_BITRATE_BPS).

Reply:
So `getStress` should return a number between 0 and 1, right?  But `MAX_TOTAL_BITRATE_BPS` is `560_000` ?

Reply:
Yeah but we scale by that max, there should be a division there: `Math.min(max, estimate) / max` is always between 0 and 1, right?

Reply:
Sorry!  I misread where the closing parenthesis was!  I wonder if it would be clearer to do:
```
double stress = (lastReportedBitrateKbps + videoStreamCountDiff * avgVideoStreamBitrateKbps) / MAX_TOTAL_BITRATE_BPS;
return Math.min(1, stress);
```